### PR TITLE
Wonder workshop blog post

### PR DESCRIPTION
**Pull request by ganicke**
Blog post content for Wonder Workshop release:

* Images are centered as controlled by css. Do we want them to the left?
* I put the two toolbox images in a table to keep them side-by-side as in the original draft.
* The blog entry for this article is not in _blog.md_. Will add that on a final push.
* The blog page is named _01-22-2018.md_. Normally these are named as release version. Can rename when version is determined.

Eventual published article link: https://makecode.com/blog/wonder-workshop/01-22-2018

### CODE REVIEW

**Comment by ganicke** [comment]
Is this still a push? This was kind of rolled into the New Year blog. @Jaqster ?

**Comment by Jaqster** [comment]
When @samelhusseini gets back next week, I was going to have him revise and author since he did most of the work here...

**Comment by ganicke** [comment]
OK, letting it idle there for now.

File: docs/blog/wonder-workshop/01-22-2018.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Face Pattern picker
+
+![Face picker](/static/blog/wonder-workshop/01-22-2018/face-picker.jpg)
+
+### Sounds menu
+
+![Sounds menu](/static/blog/wonder-workshop/01-22-2018/sound-menu.jpg)
+
+Speaking of Sounds, one thing we had a lot of fun with was using the Custom Sounds. You can record your voice saying something like, "Sam, get back to work!". Then, have Cue sneak up on people and say it! Lots of fun 😊
+
```
Comment (samelhusseini) [inline docs/blog/wonder-workshop/01-22-2018.md]: hint hint

**Comment by samelhusseini** [comment]
This looks good, I've added an entry to blog.md. What do we want to do about the date?

**Comment by pelikhan** [comment]
Are we posting this?

**Comment by ganicke** [comment]
Bueller?

**Comment by Jaqster** [comment]
It's sad to me that less then 50% of the people I talk to these days get that reference...  So, yes we should post this under Sam's name.  Sam, you want to change the date and tweak anything else about the post?

**Comment by pelikhan** [comment]
Bueller?

**Comment by pelikhan** [comment]
@ganicke can you push this blog post out?

**Pull request merged**